<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="buttons_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);
}
QPushButton{
	color:white;
	text-align:center;
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="GAN_MRI_Button">
         <property name="text">
          <string>GAN for MRI</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CNN_MRI_Button">
         <property name="text">
          <string>CNN for MRI</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>382</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="dbButton">
       <property name="text">
        <string>Database Admin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="switchSceneBtn">
       <property name="text">
        <string>USER View</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="main_widget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>1161</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>851</width>
       <height>601</height>
      </rect>
     </property>
     <widget class="QLabel" name="plotLabel_CNN">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>PLOT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>271</width>
       <height>285</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="epochs">
          <property name="text">
           <string>Epochs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_epochs">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="batch_size">
          <property name="text">
           <string>Batch size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_batch_size">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="learning_rate">
          <property name="text">
           <string>Learning rate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_learning_rate">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>181</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="folder_explore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Chose folder with data</string>
     </property>
    </widget>
    <widget class="QLabel" name="path_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>[Folder Path]</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="electrodes">
        <property name="text">
         <string>Electrodes</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEdit_electrodes">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="frequency">
        <property name="text">
         <string>Frequency</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="frame_size">
        <property name="text">
         <string>Frame size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="textEdit_frame_size">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QTextEdit" name="textEdit_frequency">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="window_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>EEG for CNN</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="info_dump">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
